test/es-module/test-require-module-tla-retry-require.js

```diff
@@ -0,0 +1,16 @@
+// This tests that after failing to require an ESM that contains TLA,
+// retrying with import() still works, and produces consistent results.
```

Comment:
Where is the assertion of success?

Reply:
Ah yeah the message should be updated to `retrying with require() still throws`..